packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -67,32 +119,30 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
   }
 
   if ([@"pickImage" isEqualToString:call.method]) {
-    _imagePickerController = [[UIImagePickerController alloc] init];
-    _imagePickerController.modalPresentationStyle = UIModalPresentationCurrentContext;
-    _imagePickerController.delegate = self;
-    _imagePickerController.mediaTypes = @[ (NSString *)kUTTypeImage ];
-
+    _phPickerFlag = NO;
```

Comment:
Do we really need this state flag? It seems like we can pass this flag along.

Reply:
I think we need this state flag. Because even the iOS version is 14 or higher we need to use the old `UIImagePickerController` when the user wants to use the camera. So I created this flag to make sure PHPicker will be used only if the user pick photo from the photo library and the iOS version is 14 or higher.